### PR TITLE
Add A11y review labels

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added the "Review: A11y" labels to the review set to indicate that the author of the change would like someone to perform an A11y specific review of it.

## QA
- Check the code looks right
- If you want to, check out this branch and follow the steps in the README to apply these labels to a dummy repo
- Check the labels have been added to the team information: https://discourse.canonical.com/t/labels-descriptions/151

### CODE REVIEW

File: labels.json [hunk 1]
```diff
@@ -8,14 +8,17 @@
...
   { "name": "Review: Design +1", "color": "#0e8420" },
   { "name": "Review: Code +1", "color": "#0e8420" },
   { "name": "Review: QA +1", "color": "#0e8420" },
   { "name": "Review: UX +1", "color": "#0e8420" },
+  { "name": "Review: A11y +1", "color": "#0e8420" },
   { "name": "Review: Design -1", "color": "#c7162b" },
   { "name": "Review: Code -1", "color": "#c7162b" },
   { "name": "Review: QA -1", "color": "#c7162b" },
   { "name": "Review: UX -1", "color": "#c7162b" },
+  { "name": "Review: Ally -1", "color": "#c7162b" },
```
Comment (petesfrench) [inline labels.json]:
```suggestion
  { "name": "Review: A11y -1", "color": "#c7162b" },
```

**Review by petesfrench — CHANGES_REQUESTED** [review]
Just a little spelling change

**Comment by anthonydillon** [comment]
Sorry missed this @petesfrench, can you update the label. I thought you were reviewing it still.

File: labels.json (hunk 1 above)
Comment (anthonydillon) [reply labels.json]: Good catch!

**Review by anthonydillon — COMMENTED** [review]
@petesfrench ready for another review

**Review by petesfrench — APPROVED** [review]
LGTM! I will make sure to updated the label next time

**Pull request merged**